plugin/src/automation/lift/lift.ml

```diff
@@ -300,6 +361,8 @@ let repack c env lifted typ sigma =
      let args = unfold_args typ in
      let sigma, typ_red = specialize_delta_f env f args sigma in
      sigma, mkLetIn (Anonymous, lifted, typ, eta_prod_rec (mkRel 1) (shift typ_red))
+  | SwapConstruct _ ->
```

Comment:
Explain: For "repack" here, it seems to have to do with the eliminators not having exactly the same structure? Will we we ever encounter this? We need to understand packing better. I guess because our eliminators are exactly the same up to renaming and swapping constructors, we have no dependent elimination?